### PR TITLE
droidcam: update to 2.1.3

### PR DESCRIPTION
**Pull request by ericktucto**
This changes are to update version of Droidcam

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my virtual maquine
  - x86_64-glibc

closes #50238

### CODE REVIEW

**Comment by classabbyamp** [comment]
commit message should be `droidcam: update to 2.1.3.`

**Comment by ericktucto** [comment]
@classabbyamp okay, I'll change commit message
Why there is a error ? [link error](https://github.com/void-linux/void-packages/actions/runs/9057447220/job/24881481152)

File: srcpkgs/droidcam/template [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+make_build_args="USBMUXD=-lusbmuxd-2.0 APPINDICATOR=ayatana-appindicator3-0.1"
 hostmakedepends="pkg-config"
 makedepends="pkg-config libjpeg-turbo-devel ffmpeg-devel alsa-lib-devel
  speex-devel libusbmuxd-devel libplist-devel gtk+3-devel
  libayatana-appindicator-devel"
-depends="v4l2loopback"
+depends="v4l2loopback linux-headers gcc make"
 short_desc="Tool for using your android device as a wireless/usb webcam"
-maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+maintainer="Erick Tucto <erick@ericktucto.com>"
```
Comment (classabbyamp) [inline srcpkgs/droidcam/template:14]:
```suggestion
maintainer="Orphaned <orphan@voidlinux.org>"
```
while the current maintainer has not contributed in a while, we'd prefer adoptions only happen when the new maintainer has some history of contributions to void, to ensure the package gets the care it needs

https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template

Comment (ericktucto) [reply srcpkgs/droidcam/template]: @classabbyamp I'm sorry, I'll change the maintainer

Comment (ericktucto) [reply srcpkgs/droidcam/template]: @classabbyamp I changed the maintainer

File: srcpkgs/droidcam/template [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 build_style=gnu-makefile
-make_build_args="USBMUXD=-lusbmuxd-2.0 JPEG=-lturbojpeg"
+make_use_env=true
+make_build_args="USBMUXD=-lusbmuxd-2.0 APPINDICATOR=ayatana-appindicator3-0.1"
 hostmakedepends="pkg-config"
 makedepends="pkg-config libjpeg-turbo-devel ffmpeg-devel alsa-lib-devel
  speex-devel libusbmuxd-devel libplist-devel gtk+3-devel
  libayatana-appindicator-devel"
-depends="v4l2loopback"
+depends="v4l2loopback linux-headers gcc make"
```
Comment (classabbyamp) [inline srcpkgs/droidcam/template:12]: why are these dependencies added?

File: srcpkgs/droidcam/template [hunk 3]
```diff
@@ -24,6 +25,7 @@ do_install() {
 	vbin droidcam
 	vbin droidcam-cli
 	vinstall icon2.png 644 usr/share/pixmaps droidcam-icon.png
+	vinstall icon2.png 644 opt droidcam-icon.png
```
Comment (classabbyamp) [inline srcpkgs/droidcam/template:28]: what's the point of this?

**Pull request merged**